tests/test-environment.ts

```diff
@@ -1,25 +1,36 @@
 import JSDOMEnvironment from 'jest-environment-jsdom';
+import fs from 'fs-extra';
+import type { TestSuiteConfig } from './utils/test-suite-config';
 
 class Environment extends JSDOMEnvironment {
     constructor(...args: ConstructorParameters<typeof JSDOMEnvironment>) {
         super(...args);
 
-        const promises = this._promises;
-        Object.defineProperty(this.global, 'waitThis', {
-            get: () => {
-                return (promise: Promise<unknown>) => {
-                    promises.push(promise);
-                };
-            },
-        });
+        const [_, context] = args;
+        this._testPath = context.testPath;
```

Comment:
where this `testPath` comes from ?

Reply:
https://github.com/jestjs/jest/blob/main/packages/jest-environment/src/index.ts#L16